dataflow/src/main/java/org/checkerframework/dataflow/cfg/builder/CFGTranslationPhaseOne.java

```diff
@@ -2559,14 +2560,20 @@ private boolean casesAreExhaustive() {
             // It's an enumerated type.
             List<VariableElement> enumConstants =
                 ElementUtils.getEnumConstants(declaredTypeElement);
-            List<Name> caseLabels = new ArrayList<>(enumConstants.size());
+            List<@Nullable Name> caseLabels = new ArrayList<>(enumConstants.size());
             for (CaseTree caseTree : caseTrees) {
               for (ExpressionTree caseEnumConstant : TreeUtils.caseTreeGetExpressions(caseTree)) {
-                caseLabels.add(((IdentifierTree) caseEnumConstant).getName());
+                if (caseEnumConstant.getKind() == Kind.NULL_LITERAL) {
+                  caseLabels.add(null);
+                } else {
+                  caseLabels.add(((IdentifierTree) caseEnumConstant).getName());
+                }
               }
             }
             // Could also check that the values match.
-            boolean result = enumConstants.size() == caseLabels.size();
+            boolean result =
+                (enumConstants.size() == caseLabels.size()
```

Comment:
This looks suspicious to me.  One of the labels could be null, in which case I think `casesAreExhaustive` should return false.  Could you write a test to assuage my concern?  Thanks.

Reply:
Oh good catch.  I don't know how to write a test case that would have caught this bug, because I can't see how a checker would behave differently when the cases are not exhaustive.